eng/pipelines/templates/stages/net-release-to-devops.yml

```diff
@@ -0,0 +1,42 @@
+parameters:
+  ShouldDownloadArtifacts: true
+  ShouldCreateGitTag: true
+  ShouldSignPackages: true
+  ShouldPublishSymbols: true
+  ShouldPublishToDevOpsFeed: true
+  ShouldPublishArtifact: true
```

Comment:
nit: While the `Should` prefix works well for reading the parameter template conditionals, I think as a parameter name it would be better to drop it, i.e. `DownloadArtifacts`, `CreateGitTag`, etc.